ftw/tabbedview/browser/resources/tabbedview.js

```diff
@@ -64,6 +64,13 @@ load_tabbedview = function(callback) {
 
     reload_view : function(callback) {
       var params = this.parse_params();
+
+      // add searchbox text property, when input is prefilled
+      // for example, when using the back button
+      if (tabbedview.searchbox.val().length >=3){
```

Comment:
The magic number 3 is now used several times, i'd add a function that defines its meaning, e.g.

```javascript
if (has_searchfield_input(tabbedview.searchbox)) {}
```